packages/generator-metal-clay/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "generator-metal-clay",
   "version": "2.1.12",
   "description": "Yeoman generator for Clay components",
-  "license": "MIT",
+  "license": "BSD",
```

Comment:
Why the license change?

Reply:
To be in line with other licensed, we thought it was wrong. Should I revert this?